causal-inference-for-the-brave-and-true/04-Graphical-Causal-Models.ipynb

```diff
@@ -490,9 +193,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As an example, let's say your knowledge of statistics causes you to know more of causal inference and machine learning. If I don't know your level of statistical knowledge, then knowing that you are good at causal inference makes it more likely that you are also good at machine learning. That is because even if I don't know your level of statistical knowledge, I can infer it from your causal inference knowledge: if you are good at causal inference you are probably good at statistics, which also makes it more likely that you are good at machine learning. \n",
+    "For example, let's say your knowledge of statistics causes you to know more about causal inference and machine learning. If I don't know your level of statistical knowledge, then knowing that you are good at causal inference makes it more likely that you are also good at machine learning. That is because even if I don't know your level of statistical knowledge, I can infer it from your causal inference knowledge. If you are good at causal inference, you are probably good at statistics, making it more likely that you are good at machine learning. \n",
     "\n",
-    "Now, if I condition on your knowledge about statistics, then how much you know about machine learning becomes independent of how much you know about causal inference. You see, knowing your level of statistics already gives me all the information I need to infer the level of your machine learning skills. Knowing your level of causal inference will give no further information in this case. \n",
+    "Now, if I condition on your knowledge about statistics, then how much you know about machine learning becomes independent of how much you know about causal inference. Knowing your level of statistics already gives me all the information I need to infer the level of your machine learning skills. Knowing your level of causal inference will provide no further information in this case. \n",
```

Comment:
I actually like the coloquial tone of "You see". 
```suggestion
    "Now, if I condition on your knowledge about statistics, then how much you know about machine learning becomes independent of how much you know about causal inference. You see, knowing your level of statistics already gives me all the information I need to infer the level of your machine learning skills. Knowing your level of causal inference will provide no further information in this case. \n",
```

Reply:
I really enjoy the casual writing style! Completely agree 👍🏽